modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java

```diff
@@ -402,6 +403,14 @@ public void testGetObjectEntries() throws Exception {
 			).build(),
 			objectEntry2, objectEntry3, objectEntry4);
 
+		testGetObjectEntries(
+			HashMapBuilder.put(
+				"filter",
+				filterString.concat(
+					buildEqualsExpressionFilterString("title", null))
```

Comment:
Hey @jairmedeiros is this causing a NPE at https://github.com/liferay/liferay-portal/blob/08475a044218ee312bc6452f6348f6d686cf8f6c/modules/apps/object/object-rest-test-util/src/main/java/com/liferay/object/rest/test/util/BaseObjectEntryManagerImplTestCase.java#L97?